plugin-flex-ts-template-v2/src/feature-library/send-audio-rec-file/custom-components/AudioRecorderPanel/AudioRecorderPanel.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import * as Flex from '@twilio/flex-ui';
+import { Actions, IconButton, templates } from '@twilio/flex-ui';
+import MicRecorder from 'mic-recorder-to-mp3';
+import { Grid, Card, Text, Paragraph } from '@twilio-paste/core';
+import { Spinner } from '@twilio-paste/core/spinner';
+
+// import { RecordIcon } from '@twilio-paste/icons/esm/RecordIcon';
+
+import { AudioRecorderWrapper, AudioRecorderPopover } from './AudioRecorderPanel.Styles';
+import { StringTemplates } from '../../flex-hooks/strings';
+
+const Mp3Recorder = new MicRecorder({ bitRate: 128 });
+
+export interface OwnProps {
+  showRecorder: boolean;
+  audioFile?: any;
+}
+
+export type Props = OwnProps;
+
+class AudioRecorderPanel extends React.Component<Props> {
+  state = {
+    isRecording: false,
+    audioURL: '',
+    blobURL: '',
+    recorder: null,
+    isBlocked: false,
+    showRecorder: this.props.showRecorder,
+    audioFile: null,
+    confirmSendFile: false,
+    recordingMessage: '',
```

Comment:
I think the `recordingMessage` state is unused and can be removed!

Reply:
you are right ... that was used before using strings.
recordingMessage state removed - Done